lib/matplotlib/ticker.py

```diff
@@ -812,15 +812,21 @@ def __init__(self, base=10.0, labelOnlyBase=False,
             base of the logarithm.
 
         labelOnlyBase : bool, optional, default: False
-            whether to only label decades ticks.
+            whether label ticks only at integer multiples of base.
+            This is normally True for major ticks and False for
+            minor ticks.
 
-        label_pruning : bool, optional, default: False
-            when set to True, label on a subset of ticks.
+        minor_thresholds : (subset, all), optional, default: (1, 0.4)
```

Comment:
-> "subset_thresholds"?  I actually had that first, then changed it to "minor" because I thought that in practice it would be closer to the actual meaning.